docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -20,6 +20,41 @@ if you want to have products data exported to Elasticsearch after `build-demo` t
 
 ### Application
 - *(low priority)* if you want to test behavior of cart with no listable product in it, implement functional test as it is in [#852](https://github.com/shopsys/shopsys/pull/852)
+- copy data fixtures into your project namespace `ShopBundle\DataFixtures` from `FrameworkBundle\DataFixtures` of [shopsys/framework] repository ([#854](https://github.com/shopsys/shopsys/pull/854))
+    - copy `PerformanceDataCommand.php` into project namespace `ShopBundle\Command` from `FrameworkBundle\Command` of [shopsys/framework] repository
+    - move configuration file `FrameworkBundle/Resources/config/services/data_fixtures.yml` to `Shopbundle/Resources/config/services/data_fixtures.yml`
+    - add new configuration file to `Shopbundle/Resources/config/services/commands.yml` and add project-base Command configuration
+    ```yaml
+    services:
+        _defaults:
+            autowire: true
+            autoconfigure: true
+            public: false
+
+        Shopsys\ShopBundle\Command\:
+            resource: '../../../Command'
+    ```
+    - import both configuration files in `services.yml`
+    ```diff
+        imports:
+            - { resource: forms.yml }
+    +       - { resource: 'services/{commands,data_fixtures}.yml' }
+    ```
+    - change namespaces of data fixtures from `Shopsys\FrameworkBundle\DataFixtures` to `Shopsys\ShopBundle\DataFixtures` (in data fixtures, tests and *.yml configurations)
```

Comment:
If I copy datafixtures from shopbundle I don't need to change namespaces in datafixtures. 
Also I'm missing some instructions on what to do when I have some datafixtures extended. Take a look at demoshop and try to imagine you have to update datafixtures there (mail templates) based on this informations only.